tsdoc-config/tsconfig.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
You don't want to pull RSC into this repo?

Reply:
It's been useful to have a counterexample of a minimalist toolchain.  I'd consider heft, though.